chain/stmgr/forks_test.go

```diff
@@ -292,14 +307,20 @@ func TestForkRefuseCall(t *testing.T) {
 		GasFeeCap:  types.NewInt(0),
 	}
 
+	nullStart := abi.ChainEpoch(testForkHeight - nullsBefore)
+	nullLength := abi.ChainEpoch(nullsBefore + nullsAfter)
+
 	for i := 0; i < 50; i++ {
```

Comment:
Not your bug: I don't love that testForkHeight is a global and that this test relies on knowing that its value is 40 in order to actually test anything.  Maybe make `testForkHeight` not a global, or change 50 to something like `2 * testForkHeight`?

Reply:
Well, being a global is intentional. But yeah, 50 being hard-coded is a problem.